samples/apps/tpcc/app/oe_sign.conf

```diff
@@ -0,0 +1,7 @@
+# Enclave settings:
+Debug=1
+NumHeapPages=50000
```

Comment:
I would increase this to something like 327680

If the total number of allocated pages are exceed then the application will just shutdown without warning

Reply:
Where does this number (327680) come from out of interest?

Reply:
its 1.3GB - I don't remember exactly we have had that for a while now